From 19a5d2b4111feae261cc7500bf0c477c4422e405 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:16:49 +0800
Subject: [PATCH] Revert "fs: partitions: Fix warnings in
 fs/partitions/check.c"

This reverts commit e27f17b5318851395a66cbaf1524ea89ff8f0cb9.
---
 fs/partitions/check.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fs/partitions/check.c b/fs/partitions/check.c
index 934637a9a6da..ecfabf53d1ca 100644
--- a/fs/partitions/check.c
+++ b/fs/partitions/check.c
@@ -327,8 +327,9 @@ static void part_release(struct device *dev)
 	kfree(p);
 }
 
-static int part_uevent(struct device *dev, struct kobj_uevent_env *env)
+static int part_uevent(struct device *dev, struct kobj_uvent_env *env)
 {
+	struct gendisk *disk = dev_to_disk(dev);
 	struct hd_struct *part = dev_to_part(dev);
 
 	add_uevent_var(env, "PARTN=%u", part->partno);
-- 
2.35.3

